Block/Checkout/Success.php

```diff
@@ -204,4 +169,16 @@ public function getOrder()
         return $this->order;
     }
 
+    /**
+     * @throws NoSuchEntityException
+     */
+    public function getMaskedQuoteId(int $quoteId): ?string
```

Comment:
Why accepting a method argument here and not just call `->getOrder()->getQuoteId()` from this method?